Updates-2.0/u2-sce-test-wso2ei-6.5.0-full-testgrid.yaml

```diff
@@ -0,0 +1,62 @@
+# TestGrid job configuration
+
+version: '0.9'
+emailToList: "builder@wso2.org,chaminda@wso2.com,isuruu@wso2.com,nandika@wso2.com,kasuna@wso2.com,
+              milindap@wso2.com,dulanjali@wso2.com,sasikala@wso2.com,isuruuy@wso2.com,
```

Comment:
Keep the following emails and remove the rest please.

`integration-builder@wso2.com,chanika@wso2.com,dilini@wso2.com`

Reply:
Done